features/step_definitions/supplier_steps.rb

```diff
@@ -13,16 +13,16 @@
   response = call_api(:get, "/frameworks")
   expect(response.code).to eq(200), _error(response, "Failed getting frameworks")
   frameworks = JSON.parse(response.body)['frameworks']
-  frameworks.delete_if {|framework| not ['coming', 'open'].include?(framework['status'])}
+  frameworks.delete_if {|framework| not %w[coming open].include?(framework['status'])}
   if frameworks.length > 1
     puts 'SKIPPING as there is more than one framework coming or open'
     skip_this_scenario
   end
 end
 
 Given /^that(?: (micro|small|medium|large))? supplier has applied to be on that framework$/ do |organisation_size|
-  organisation_size ||= ['micro', 'small', 'medium', 'large'].sample
-  submit_supplier_declaration(@framework['slug'], @supplier["id"], 'status': 'complete', 'organisationSize': organisation_size, 'nameOfOrganisation': 'foobarbaz', 'primaryContactEmail': 'foo@bar.com')
+  organisation_size ||= %w[micro small medium large].sample
+  submit_supplier_declaration(@framework['slug'], @supplier["id"], {'status': 'complete', 'organisationSize': organisation_size, 'nameOfOrganisation': 'foobarbaz', 'primaryContactEmail': 'foo@bar.com'})
```

Comment:
I thought the curly braces in the last arg weren't needed - we took them out of this line in the last PR?

Reply:
Oops! Rebase error. Fixed! :)